templates/components/colors.html

```diff
@@ -0,0 +1,81 @@
+{% block colors-content %}
+
+<h3>Color usage</h3>
+
+{% block colors-intro %}
```

Comment:
yeah I was doing that and then it made the compiled templates even stupider, but I think you're probably right and readability on the files you're working on is more important